components/brave_rewards/resources/extension/brave_rewards/utils.ts

```diff
@@ -41,7 +41,8 @@ export const generatePromotions = (promotions?: RewardsExtension.Promotion[]) =>
   }
 
   let claimedPromotions = promotions.filter((promotion: Rewards.Promotion) => {
-    return promotion.status === 4 // PromotionStatus::FINISHED
+    return promotion.status === 4 || // PromotionStatus::FINISHED
+      promotion.status === 5 // PromotionStatus::OVER
```

Comment:
nit: @NejcZdovc not for this PR as unrelated, but `::OVER` would be better named `EXPIRED`

Reply:
@NejcZdovc Business logic like this should be moved to the libs (but not for this ticket as unrelated)

Reply:
see comment above